crates/core/src/models/signature.rs

```diff
@@ -0,0 +1,33 @@
+use reth_rpc_types::Signature;
```

Comment:
can we alias this as Eth_Signature? to make it very clear that it's an Eth signature